### DISCUSSION TITLE
speaker diarization❓ Questions / Help / Support

### DISCUSSION THREAD

**Discussion by jainal09** [snakers4/silero-models#16] [General]
## ❓ Questions and Help

Hi,
I wanted to know i want to perform multi speaker transcript with multiple speakers and their speech to text data.
Is it possible with silero-models or are you going to add such feature?

**Comment by snakers4** [comment]
Hi,

We have this feature in the EE (-version).
We are not planning to add this in near future to CE models.

PS
But I believe in real life in production it is just easier to ask to write stereo.

**Comment by jainal09** [comment]
>But I believe in real life in production it is just easier to ask to write stereo.

I didnt understand this.
What do you mean by this?

**Comment by snakers4** [comment]
If you record 2 channels you do not need diarization.

**Comment by jainal09** [comment]
Hi, i have a situation where 5 people are speaking and sometimes conversations can be overlapped meaning 2 speakers can speak simultaneously. What to do here?

**Comment by snakers4** [comment]
Hi, 

In the simplest case - you can just see an example with TorchAudio VAD and try first splitting audio using VAD.

Usually overlapping segments are a small percentage of audio.
